### PR TITLE
DIV-5103 - Update email address for divorce cases

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

[DIV-5103 - Update email address for divorce cases](https://tools.hmcts.net/jira/browse/DIV-5103)
 - change contactdivorce@justice.gov.uk to divorcecase@justice.gov.uk

### CODE REVIEW

File: infrastructure/variables.tf [hunk 1]
```diff
@@ -394,7 +394,7 @@ variable "court_service_centre_openinghours" {
 }
 
 variable "court_service_centre_email" {
-  default = "contactdivorce@justice.gov.uk"
+  default = "divorcecase@justice.gov.uk"
```
Comment (qzhou-hmcts) [inline infrastructure/variables.tf]: You need to update aks config to see this see  changes on preview environment .
charts/div-pfe/values.yaml

Comment (dtaylor7) [reply infrastructure/variables.tf]: my understanding was defaults get set in case orch service?

Comment (qzhou-hmcts) [reply infrastructure/variables.tf]: variables.tf only applies to terraform, however aks chart config does not go through terraform

Comment (dtaylor7) [reply infrastructure/variables.tf]: have updated, but if case orch service sets it wont make a difference

File: charts/div-pfe/values.yaml [hunk 2]
```diff
@@ -23,7 +23,7 @@ nodejs:
         COURT_SERVICE_CENTRE_CITY: "Nottingham"
         COURT_SERVICE_CENTRE_POBOX: "PO Box 10447"
         COURT_SERVICE_CENTRE_POSTCODE: "NG2 9QN"
-        COURT_SERVICE_CENTRE_EMAIL: "contactdivorce@justice.gov.uk"
```
Comment (yarpo) [inline charts/div-pfe/values.yaml]: What about line 19:
```
        COURT_EMAIL: "contactdivorce@justice.gov.uk"
```

Comment (dtaylor7) [reply charts/div-pfe/values.yaml]: only use the new email once case is submitted

File: infrastructure/variables.tf [hunk 3]
```diff
@@ -394,7 +394,7 @@ variable "court_service_centre_openinghours" {
 }
 
 variable "court_service_centre_email" {
```
Comment (yarpo) [inline infrastructure/variables.tf]: what about line 416:
```
variable "court_email" {
  default = "contactdivorce@justice.gov.uk"
}
```

Comment (dtaylor7) [reply infrastructure/variables.tf]: same as https://github.com/hmcts/div-petitioner-frontend/pull/563#discussion_r299089941

File: charts/div-pfe/values.yaml (hunk 2 above)
Comment (yarpo) [reply charts/div-pfe/values.yaml]: oh, ok. Sorry. My mistake then

**Comment by dtaylor7** [comment]
@yarpo 
> Can you please also update:
> div-petitioner-frontend/test/end-to-end/data/amendPetitionSession.js:122
Already done

> div-petitioner-frontend/bin/run-functional-tests.sh
this needs to stay the same as its used before the case is created

**Pull request merged**